src/k8s/clone.js

```diff
@@ -58,17 +62,24 @@ const cleanVm = vm => {
     delete vm.spec.dataVolumeTemplates;
   }
 
+  if (has(vm.spec, 'template.spec.domain.firmware')) {
+    delete vm.spec.template.spec.domain.firmware;
+  }
+
   delete vm.status;
 };
 
-const updateVm = (vm, name, namespace, description, startVm) => {
+const updateVm = (vm, name, namespace, description, startVm, osName) => {
   vm.metadata.name = name;
   vm.metadata.namespace = namespace;
   vm.metadata.annotations = {
-    ...(vm.metadata.annotations || {}),
```

Comment:
the annotation could be used for many things. What is the main reason for deleting them? 

Wouldn't it be worth it to give the user the choice to keep some of them?